<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxA">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>15.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxB">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>15.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxC">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>15.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DrawerWidget" name="glWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawerWidget</class>
   <extends>QWidget</extends>
   <header>drawerwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxA</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Dialog</receiver>
   <slot>abcChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxB</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Dialog</receiver>
   <slot>abcChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxC</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Dialog</receiver>
   <slot>abcChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>abcChanged()</slot>
 </slots>
</ui>
